ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/sflow_settings.schema.yml

```
@@ -0,0 +1,48 @@
+# Copyright (c) 2023 Arista Networks, Inc.
+# Use of this source code is governed by the Apache License 2.0
+# that can be found in the LICENSE file.
+# yaml-language-server: $schema=../../../../plugins/plugin_utils/schema/avd_meta_schema.json
+# Line above is used by RedHat's YAML Schema vscode extension
+# Use Ctrl + Space to get suggestions for every field. Autocomplete will pop up after typing 2 letters.
+type: dict
+keys:
+  sflow_settings:
+    documentation_options:
+      table: management-sflow-settings
+    type: dict
+    description: |-
+      sFlow settings.
+      The sFlow process will only be configured if any interface is enabled for sFlow.
+      For default enabling of sFlow for various interface types across the fabric see "fabric_sflow".
```

in `fabric_sflow.schema.yml` we use backticks rather than double quotes but not sure we have a good guideline on this not consistency

```suggestion
      For default enabling of sFlow for various interface types across the fabric see `fabric_sflow`.
```